CondTools/Ecal/plugins/EcalPedestalHistory.h

```diff
@@ -41,12 +41,12 @@
 #include <fstream>
 #include <map>
 
-class EcalPedestalHistory : public edm::EDAnalyzer {
+class EcalPedestalHistory : public edm::one::EDAnalyzer<> {
 public:
   explicit EcalPedestalHistory(const edm::ParameterSet&);
   ~EcalPedestalHistory() override;
   void analyze(const edm::Event&, const edm::EventSetup&) override;
-  void beginRun(edm::Run const&, edm::EventSetup const&) override;
+  void beginRun(edm::Run const&, edm::EventSetup const&);
```

Comment:
You need to make a `edm::one::EDAnalyzer<edm::one::WatchRuns>` if you want to use `beginRun()`. And keep `override` then.

Reply:
updated